docs/docusaurus/scripts/remark-named-snippets/snippet.js

```diff
@@ -173,7 +173,7 @@ function getDirs() {
   return manualDirs.concat(versionDirs)
 }
 
-function main() {
+function main(verbose = false) {
```

Comment:
Where would we (if ever) run this function as `verbose = true`?

Reply:
If so, how would we do this?

Reply:
Good call, in https://github.com/great-expectations/great_expectations/pull/7894/commits/19b2b1f9a76f64e6182196b5fe293a0cb168c9c6 I changed `yarn snippet-check` to use the verbose mode which you can run locally to check all snippets.